3ds Max/Max2Babylon/Exporter/BabylonExporter.IMaterialExporter.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.Drawing;
+using Autodesk.Max;
+using BabylonExport.Entities;
+using GLTFExport.Entities;
+
+namespace Max2Babylon
+{
+    
+
+    public interface IMaterialExporter
+    {
+        ClassIDWrapper MaterialClassID { get; }
+    }
+
+    public interface IBabylonMaterialExporter : IMaterialExporter
+    {
+        BabylonMaterial ExportBabylonMaterial(IIGameMaterial material);
+    }
+
+    delegate string TryWriteImageCallback(string sourceTexturePath);
```

Comment:
Do we ever use this delegate?

Reply:
this is not used inside babylon but we use it to export image from our custom material exporter